packages/lab/src/__tests__/__e2e__/tokenized-input-next/TokenizedInputNext.cy.tsx

```diff
@@ -0,0 +1,201 @@
+import { composeStories } from "@storybook/react";
+import * as tokenizedInputNextStories from "@stories/tokenized-input-next/tokenized-input-next.stories";
+import { checkAccessibility } from "../../../../../../cypress/tests/checkAccessibility";
+import { MultilineInput } from "@salt-ds/core";
+import { ChangeEvent, useState } from "react";
+
+const statesData = [
+  "Alabama",
+  "Alaska",
+  "Arizona",
+  "Arkansas",
+  "California",
+  "Colorado",
+  "Connecticut",
+  "Delaware",
+  "Florida",
+  "Georgia",
+  "Hawaii",
+  "Idaho",
+  "Illinois",
+  "Indiana",
+  "Iowa",
+  "Kansas",
+  "Kentucky",
+  "Louisiana",
+  "Maine",
+  "Maryland",
+  "Massachusetts",
+  "Michigan",
+  "Minnesota",
+  "Mississippi",
+];
+
+const composedStories = composeStories(tokenizedInputNextStories);
+
+const { Default } = composedStories;
+
+describe("GIVEN a Tokenized Input", () => {
+  checkAccessibility(composedStories);
+  describe("WHEN disabled", () => {
+    it("SHOULD mount as disabled", () => {
+      cy.mount(<Default initialSelectedItems={statesData} disabled />);
+      cy.findByRole("textbox").should("be.disabled");
+    });
+  });
+  describe("WHEN mounted as an uncontrolled component", () => {
+    it("should render the Tokenized Input with pre selected items", () => {
+      cy.mount(<Default initialSelectedItems={statesData} />);
```

Comment:
Why are we passing data into stories like this? The stories should come with their own data and we can test it here